standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/common/histogram/kll/KllUtils.java

```diff
@@ -100,7 +101,7 @@ public static KllFloatsSketch deserializeKll(final byte[] buf) {
    * @return the length of the given KLL sketch according to the given java data model
    */
   public static int lengthFor(JavaDataModel model, KllFloatsSketch kll) {
-    return model == null ? KllFloatsSketch.getMaxSerializedSizeBytes(kll.getK(), kll.getN())
+    return model == null ? KllFloatsSketch.getMaxSerializedSizeBytes(kll.getK(), kll.getN(), KllSketch.SketchType.FLOATS_SKETCH, false)
```

Comment:
In this commit: https://github.com/apache/datasketches-java/commit/60cbfaabd43ab33eb542148eb4dbd8fbef98a264#diff-dacbef77551b8e94e3095811486b859725948654627b5141625c453f0ecad774

`KllFloatsSketch::getMaxSerializedSizeBytes` added two extra parameters:
![method_change](https://github.com/user-attachments/assets/ecd2c406-762d-4820-b06f-3cc18b50aa20)

I set the updatable to false to keep it the same as before. If we want optimize this to set true or false, we can make a configuration here in the future.